web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -560,6 +560,50 @@ describe("<BusinessFormationPaginator />", () => {
         expect(screen.queryByText(Config.formation.errorBanner.errorOnStep)).not.toBeInTheDocument();
         expect(page.getStepStateInStepper(LookupStepIndexByName("Billing"))).toEqual("COMPLETE-ACTIVE");
       });
+
+      it("shows the Trustees label when members' field has an error and legalType is nonprofit", async () => {
+        const nonprofit = generateBusiness({
+          formationData: generateFormationData({
+            formationFormData: generateFormationFormData({
+              members: undefined,
+            }),
+          }),
+          profileData: generateProfileData({ legalStructureId: "nonprofit" }),
+        });
+        const page = preparePage({ business: nonprofit, displayContent });
+
+        await page.stepperClickToReviewStep();
+        await page.submitReviewStep();
+
+        await page.stepperClickToContactsStep();
+        expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.errorBanner.errorOnStep);
+        expect(screen.queryByRole("alert")).not.toHaveTextContent(Config.formation.fields.members.label);
+        expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.fields.trustees.label);
+      });
+
+      const legalTypes = ["c-corporation", "s-corporation"];
+
+      it.each(legalTypes)(
+        "shows Board of Directors label when members' field has an error and legalType is %s",
+        async (legalType) => {
+          const corporation = generateBusiness({
+            formationData: generateFormationData({
+              formationFormData: generateFormationFormData({ members: undefined }),
+            }),
+            profileData: generateProfileData({ legalStructureId: legalType }),
+          });
+          const page = preparePage({ business: corporation, displayContent });
+
+          await page.stepperClickToReviewStep();
+          await page.submitReviewStep();
+
+          await page.stepperClickToContactsStep();
+
+          expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.errorBanner.errorOnStep);
+          expect(screen.queryByRole("alert")).not.toHaveTextContent(Config.formation.fields.members.label);
+          expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.fields.directors.label);
+        }
+      );
```

Comment:
It feels like there should be an extra test here? Like one for looking of the "directors" text in addition to the "trustees" text

Reply:
@adidner: Corps have Board of Directors (BOD), while NPs have Board of Trustees so the BOD label would/should never exist for the NP type. Lmk if I misunderstood your suggestion.